packages/website-static/media.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Patua+One|Roboto+Condensed:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="media.css">
+</head>
+
+<body>
+    <div id="root">
+        <div class="text-center jumbotron">
+            <h1><a class="home-link" href="index.html">xMacroscope</a></h1>
+            <div class="justify-content-center row">
+                <div class="col-8 col-lg-6">
+                    <p class="lead">xMacroscope is a <strong>design</strong>, <strong>build</strong>,
+                        <strong>research</strong> project to study how visitors at science centers understand data
+                        visualization techniques.
+                    </p>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col">
+                    <ul class="justify-content-center nav">
+                        <li class="nav-item"><a class="nav-link " href="index.html" aria-current="page">Home</a></li>
+                        <li class="nav-item"><a class="nav-link" href="research.html">Research</a></li>
+                        <li class="nav-item"><a class="nav-link active" href="">Media</a></li>
+                        <li class="nav-item"><a class="nav-link" href="https://xmacroscope.org/demo/home"
```

Comment:
Make links relative for when they are deployed. So, just remove "https://xmacroscope.org/" for the demo and site visualizations links.